block/retriever.go

```diff
@@ -38,6 +38,7 @@ func (m *Manager) RetrieveLoop(ctx context.Context) {
 // the actual blocks from the DA.
 func (m *Manager) syncUntilTarget(targetHeight uint64) error {
 	for currH := m.State.Height(); currH < targetHeight; currH = m.State.Height() {
+		m.logger.Info("Syncing from DA", "store height", currH, "target height", targetHeight)
```

Comment:
please put info logs only **after** the thing that happened

so change to debug or moved to end of for loop and put `Synced`

Reply:
done